scss/footer.scss

```diff
@@ -0,0 +1,26 @@
+.foot {
+  align-items: center;
+  background: $blue;
+  display: flex;
+  flex-direction: column;
+  height: 150px;
```

Comment:
I'd try not to make footers not so rigid, since sometimes they can be extended by additional info. Can you try maybe achieving the proposed design without setting a fixed height? 🙂